esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3760,13 +3770,20 @@ int CWsWuFileHelper::zipAFolder(const char* folder, bool gzip, const char* zipFi
     return (system(zipCommand.str()));
 }
 
+#ifdef _CONTAINERIZED
```

Comment:
We discussed this in our triage meeting.  People seem to agree that keeping the extra variable is probably worth it to greatly simplify the containerized IFDEFs. Rather than changing the function parameter list, and maybe even object variable just make it so that variable isn't used or has a value that makes it obsolete. That affects several similar changes in this PR.